### DISCUSSION TITLE
How are the nft challenges after every section meant to be solved?

### DISCUSSION THREAD

**Discussion by breellz** [Cyfrin/foundry-full-course-cu#2494] [Q&A] [answered]
On clicking the Sepolia or zksync link, i just get redirected to to the smart contract on etherscan. Where do I see the challenges to solve?

**Comment by EngrPips** [comment, marked as answer]
Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018

**Reply by breellz** [reply]
Thank you

**Reply by EngrPips** [reply]
You are welcome, Friend.
